experimental/go-dep/image/Dockerfile-stack

```diff
@@ -0,0 +1,38 @@
+FROM golang:1
+
+ENV APPSODY_MOUNTS=/:/project/user-app
+
+ENV APPSODY_WATCH_DIR=/project/user-app
+ENV APPSODY_WATCH_IGNORE_DIR=/project/user-app/vendor
+ENV APPSODY_WATCH_REGEX="^.*.(go|toml)$"
+
+ENV APPSODY_INSTALL="bash /vendor.sh"
```

Comment:
can you rename this environment variable to `APPSODY_PREP` as `APPSODY_INSTALL` is deprecated - as documented in https://appsody.dev/docs/stacks/environment-variables